### PR TITLE
use SymfonyStyle for command output

### PR DESCRIPTION
**Pull request by OskarStark**
Refs https://github.com/doctrine/doctrine2/pull/6655

screenshot 2017-12-01 15 57 59

screenshot 2017-12-01 16 10 42

### CODE REVIEW

**Comment by OskarStark** [comment]
Please review commit by commit

**Comment by ruudk** [comment]
Can you rebase on master to let the tests pass? Master was broken.

**Comment by OskarStark** [comment]
> Can you rebase on master to let the tests pass? Master was broken.

I rebased the branch 👍 

If this gets merged, I will do the same for the migrations bundle

**Comment by OskarStark** [comment]
could you please merge this PR or give feedback, if there is anything I should change?

thank you

**Comment by OskarStark** [comment]
ping

**Comment by ruudk** [comment]
It's up to @alcaeus to merge this :)

**Comment by alcaeus** [comment]
I can look at this in a few days, maybe one of the Symfony guys care to chime in? Ping @weaverryan @fabpot

**Comment by OskarStark** [comment]
> I can look at this in a few days, maybe one of the Symfony guys care to chime in? Ping @weaverryan @fabpot

@alcaeus no stress, I just wanted to know, you realized this PR 😄

File: Command/LoadDataFixturesDoctrineCommand.php [hunk 1]
```diff
@@ -56,68 +57,58 @@ protected function configure()
...
 
         if ($input->getOption('shard')) {
             if (!$em->getConnection() instanceof PoolingShardConnection) {
-                throw new \LogicException(sprintf("Connection of EntityManager '%s' must implement shards configuration.", $input->getOption('em')));
+                $ui->error(sprintf(
+                    'Connection of EntityManager "%s" must implement shards configuration.',
+                    $input->getOption('em')
+                ));
+
+                return 1;
```
Comment (weaverryan) [inline Command/LoadDataFixturesDoctrineCommand.php]: I like the pretty errors that `$ui->error()` gives us, but I'd like to only use it for "normal errors" - error that happen as a result of somewhat "normal" use of the command. So, I'd prefer to keep the `LogicException` here (but I'm ok with the error below for "Could not find any fixture services to load")

Comment (OskarStark) [reply Command/LoadDataFixturesDoctrineCommand.php]: Ok 👍

**Comment by OskarStark** [comment]
Done and rebased @weaverryan 👍

**Comment by alcaeus** [comment]
Thanks @OskarStark!

**Pull request merged**